plugins/basic/nexus-repository-raw/src/main/java/org/sonatype/nexus/repository/raw/internal/RawContentFacetImpl.java

```diff
@@ -110,8 +111,17 @@ public Void execute(final StorageTx tx) {
               asset = tx.firstAsset(component);
             }
 
-            asset.formatAttributes().set(P_LAST_VERIFIED_DATE, new Date());
-            tx.setBlob(asset, path, streamSupplier.get(), hashAlgorithms, null, content.getContentType());
+            if (payload instanceof Content) {
```

Comment:
I think this, too, is basically a logical duplicate, just masked by the slightly different formulation. But the basic idea is the same. Actually, does this version accidentally omit setting LAST_MODIFIED if the content attributes don't include it?  The others did that.